start_control_ssm.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+log() {
+  echo "$*" >&2
+}
+
+enable_hybrid_env_ssm() {
+  # SSM parameters necessary to register hybrid environment Linux
+  failed_ssm_params=0
+  activation_code=$(get_user_data_ssm "activation-code")
+  activation_id=$(get_user_data_ssm "activation-id")
+  region=$(get_user_data_ssm "region")
+
+  # If any of the parameters are missing, then exit 1.
+  if [[ "${failed_ssm_params}" -ne 0 ]]; then
```

Comment:
Maybe I'm bad at shell, but how could this ever be anything but zero? Did you mean to check an exit code or something?

Reply:
The logic is a bit nested, but it's incremented in `get_user_data_ssm`. Because of that I didn't make it a local variable. This was out of a desire to not rely on `set -e`.

Reply:
I'd rather not create a variable here inside a function that gets treated like a contstant (even though bash is fine with this).  I think rather than keeping a count, we can assume that all parameters are needed and fail if any of them aren't found.